migration/migration3.go

```diff
@@ -19,7 +20,7 @@ type storageJob struct {
 // V3StorageJobsIndexMigration contains the logic to upgrade a datastore from
 // version 2 to version 3.
 var V3StorageJobsIndexMigration = Migration{
-	UseTxn: true,
+	UseTxn: false,
```

Comment:
Since this is parallelized now, it can't run in a txn. It was also quite slow to run it serially on big hosted-pows (e.g: landsatdata).